Purchases/RCCrossPlatformSupport.h

```diff
@@ -19,7 +19,7 @@
 #endif
 
 #if TARGET_OS_MACCATALYST
-#define PLATFORM_HEADER @"uikitformac"
+#define PLATFORM_HEADER @"iOS" // temporarily until backend support is added for catalyst platform
```

Comment:
I am a little bit confused about this change. The backend will send the iOS products, but those don't work when running the app in Catalyst mode 🤔

Reply:
the backend will decide whether to use the Universal store or the Mac app store depending on the app, using a flag called `uses_legacy_mac_app_store`. 
See https://github.com/RevenueCat/khepri/pull/1253
The idea is that once Universal Purchases are a thing, Catalyst apps will default to using the iOS store, and the rest should work

Reply:
well, actually, note that this particular change is just temporary until the backend is deployed. Once that happens, we'll go back to the behavior I just described

Reply:
in this instance, I set it to iOS, since I wanted the universal app store, to check that this was working with Universal Purchases